src/push.js

```diff
@@ -16,7 +15,7 @@ export default () => {
   const pushOutput = execSync(
     `${bin} push --access-token $PHRASEAPP_TOKEN --wait`
   ).toString()
-  console.log(pushOutput)
+  console.info(pushOutput)
```

Comment:
well, that's an interesting way to get around `no-console` :D

Reply:
it's actually configured to allow info, warning and error in the linter package